level1.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
```

Comment:
You can complete your game without split files of `html`. I believe you will figure that out. Focus on reducing repetition.